agent/tool-scripts/prometheus-metrics

```diff
@@ -168,30 +164,40 @@ case "$mode" in
 	;;
 	start)
 	mkdir -p $tool_output_dir/json
-	while read -u 9 master; do
-		# fetch keys from openshift master and set default values for cert and key vars if not defined
+	while read -u 9 file; do
+		# read the input file for host, port, cert and key if any. In case the file doesn't have certs, port and they 
+		# are not defined while registering the tool, the default values are set.
+		# host cert=<cert> key=<key> port=<port>
```

Comment:
This format does not agree with the calculations you do in the next four lines. Which is right?